baseui/templates/baseui/footer.html

```diff
@@ -0,0 +1,20 @@
+<div class="columns">
+  <div class="column">
+    <ul class="nav-spaced">
+      <li><a href="/help/contact">Contact</a></li>
+      <li><a href="https://twitter.com/arxiv">Find us on Twitter</a></li>
+    </ul>
+  </div>
+  <div class="column">
+    <ul class="nav-spaced">
+      <li><a href="https://blogs.cornell.edu/arxiv/">Blog - the latest news</a></li>
+      <li><a href="https://confluence.cornell.edu/display/arxivpub/arXiv+Public+Wiki">Wiki - project documentation</a></li>
+    </ul>
+  </div>
+  <div class="column">
+    <ul class="nav-spaced">
+      <li><a href="mailto:web-accessibility@cornell.edu">Web Accessibility Help</a></li>
+      <li><a href="/help">Help with using arXiv</a></li>
+    </ul>
+  </div>
+</div>
```

Comment:
Seeing quite a few things here that might belong in a configuration file (email addresses, blog URL, etc). Something to think about for the future.

Reply:
Yes, I agree on the config file - also think that can happen in the future when we know more about what we need to have here (this footer is pretty much untested and isn't quite what we have now).